packages/components/src/dropdown-menu-v2-ariakit/styles.ts

```diff
@@ -71,7 +75,13 @@ export const DropdownMenu = styled( Ariakit.Menu )<
 	/* TODO: is there a way to read the sass variable? */
 	z-index: 1000000;
 
-	min-width: 220px;
+	display: grid;
+	grid-template-columns: minmax( 0, max-content ) 1fr;
```

Comment:
This is where the main grid layout is defined. The first column is for the prefix (and can collapse to 0 width if no prefixes are rendered within the same dropdownmenu popover).